Makefile.build

```diff
@@ -197,30 +141,22 @@ build-manifests: build-manifests-operator build-manifests-oss
 .PHONY: build-manifests-oss
 build-manifests-oss: "$(GOBIN)/addlicense" "$(BIN_DIR)/kustomize" $(OUTPUT_DIR)
 	@ echo "+++ Generating manifests in $(OSS_MANIFEST_STAGING_DIR)"
-	@ echo "    Using these tags:"
-	@ echo "    $(RECONCILER_MANAGER_IMAGE): $(RECONCILER_MANAGER_TAG)"
-	@ echo "    $(HYDRATION_CONTROLLER_IMAGE): $(HYDRATION_CONTROLLER_TAG)"
-	@ echo "    $(HYDRATION_CONTROLLER_WITH_SHELL_IMAGE): $(HYDRATION_CONTROLLER_WITH_SHELL_TAG)"
-	@ echo "    $(RECONCILER_IMAGE): $(RECONCILER_TAG)"
-	@ echo "    $(ADMISSION_WEBHOOK_IMAGE): $(ADMISSION_WEBHOOK_TAG)"
-	@ echo "    $(OCI_SYNC_IMAGE): $(OCI_SYNC_TAG)"
-	@ echo "    $(HELM_SYNC_IMAGE): $(HELM_SYNC_TAG)"
-	@ echo "    $(ASKPASS_IMAGE): $(ASKPASS_TAG)"
+	@ echo " Using tags: $(REGISTRY):*/$(IMAGE_TAG)"
```

Comment:
nit:`$(REGISTRY)/*:$(IMAGE_TAG)"`, the oder of `:` and `/` should be swapped.

Reply:
good catch - fixed!